File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-05/plagiarized/L4/09/Level4.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/

/**
 *
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
object Level4 {
    @JvmStatic
    fun main(args: Array<String>) {
        val masukkan = Scanner(System.`in`)
        var number: Int
        var sisaBagi: Int

        print("Enter an integer: ")
        number = masukkan.nextInt()

        while (number != 0) {
            sisaBagi = number % 10
            print(sisaBagi)
            number = number / 10
        }
        println()
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L4/09/Level4.ts
import { Scanner } from 'readline-sync';

function main(): void {
  const masukkan: Scanner = new Scanner();
  let number: number;
  let sisaBagi: number;

  console.log('Enter an integer: ');
  number = masukkan.nextInt();

  while (number !== 0) {
    sisaBagi = number % 10;
    console.log(sisaBagi);
    number = Math.floor(number / 10);
  }
  console.log();
}

main();

